ambry-api/src/test/java/com.github.ambry/account/AccountContainerTest.java

```diff
@@ -468,6 +468,31 @@ public void testRemoveNonExistContainer() throws JSONException {
   }
 
   /**
+   * Tests for {@link Account#UNKNOWN_ACCOUNT} and {@link Container#UNKNOWN_CONTAINER}.
+   */
+  @Test
+  public void testUnknownAccountAndContainer() {
```

Comment:
do we need a test for this? My vote would be for no - this is just java object construction - no functionality.